src/reducers/SelectedRevisionsSlice.ts

```diff
@@ -28,8 +28,16 @@ const selectedRevisions = createSlice({
       state,
       action: PayloadAction<{
         selectedRevisions: RevisionsList[];
+        isBaseDeletion: boolean;
       }>,
     ) {
+      if (action.payload.isBaseDeletion) {
+        return {
+          ...state,
+          revisions: action.payload.selectedRevisions.slice(1),
```

Comment:
I think it's more correct to slice in the dispatch call rather than here. So that this code block looks more similar to the code below for deleting new revisions. What do you think?

Reply:
Yes, we do store both in the same array and it's not great. I'm glad that will get simpler with your patch. Yeah, I'll slice in the dispatch. And when you land your patch, we can update accordingly if needed.